### PR TITLE
6 icons: AVR-Remote, lichess, Print, Location Share, Notification notes, MJ PDF Reader

### PR DESCRIPTION
**Pull request by Aircan**
First contribution. Hope that fits. Otherwise please let me know.
.svg files in /others and appfilters.xml file appended accordingly.

### CODE REVIEW

**Comment by Donnnno** [comment]
Hi, thanks for your contribution, the icons look good!
A couple of things:
- we already have one for lichess
- can you make them white? #fff
- make sure that every line-width is 1px
- remove clipping masks from your icons

**Comment by Aircan** [comment]
- lichess: Not part of my (latest) version, hence created. Shall I remove it from my repo., or will you "ignore"?
- white: Will do so
- 1 px: Double checked this (I thought). Which icon is affected?
- clipping: Which one is affected? Hints on how to?

After rework I will open another PR.

**Comment by Donnnno** [comment]
You can just update this one ;) No need to create a new PR for some quick fixes

**Comment by Aircan** [comment]
Updated accordingly, please check if your 4 bullet points are resolved.
Kept lichess in, sure you will ignore.

**Comment by Aircan** [comment]
@Donnnno I was almost creating an icon for LibreTube, since this is not available on my device, although having installed <latest>. But in appfilter.xml the actual app info is entered here. So, a bug? Or part of main branch, but not yet released/delivered?

**Comment by Donnnno** [comment]
I'll take a look

**Comment by Aircan** [comment]
Removed lichess icon and updated entry in appfilter.xml. Obviously there was a name change.
What is the conflict about, that need to be manually solved? Next time I can then consider that instantly? Is it about alphabetical order?

**Comment by Aircan** [comment]
@Donnnno anything left open, that hinders the merge?
When do you plan the next release?

File: other/appfilter.xml [hunk 1]
```diff
@@ -7024,7 +7041,7 @@
     <!-- Life Fits Into Home -->
     <item component="ComponentInfo{tr.gov.saglik.hayatevesigar/com.turktelekom.guvenlekal.ui.activity.SplashActivity}" drawable="life_fits_into_home" />
 
-    <!-- Liftago -->
```
Comment (Donnnno) [inline other/appfilter.xml:7027]: can you re-add the > here?

**Comment by Donnnno** [comment]
Hi, I hope to create a new release soon. I'm pretty busy with lots of different stuff, but after that, I have time to update it.

File: other/appfilter.xml (hunk 1 above)
Comment (Aircan) [reply other/appfilter.xml:7027]: me? In my forked appfilter.xml, everything looks fine without missing >

**Comment by Donnnno** [comment]
Hi, I had to remake all the icons.

I noticed there were images embedded, there elements were erased or something? 

image

And had to change the names because there are no capitals or spaces allowed

**Pull request merged**